versions.json

```diff
@@ -1,5 +1,6 @@
 {
   "design-system": [
+    "8.0.0",
     "8.0.0-beta.3",
```

Comment:
do we remove all these betas?

Reply:
I'll make an update to the release script to do this automatically in the future, and I'll remove them in the release branch for now